src/scripts/main.js

```diff
@@ -21,7 +21,26 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  function buildList(obj) {
+    const ul = document.createElement('ul');
+
+    for (const key in obj) {
+      const li = document.createElement('li');
+      li.textContent = key;
+
+
+      if (Object.keys(obj[key]).length > 0) {
+        li.appendChild(buildList(obj[key]));
+      }
+
+      ul.appendChild(li);
+    }
```

Comment:
You are using a `for...in` loop to iterate over the properties of an object. While this works, it's important to note that `for...in` will also iterate over inherited properties. To avoid any unexpected behavior, it's recommended to use `Object.prototype.hasOwnProperty` to check if the property belongs to the object itself. Example: `if (obj.hasOwnProperty(key)) {...}`